docs/process/teams.md

```diff
@@ -0,0 +1,12 @@
+These teams are not set in stone and will likely evolve as we continue to iterate on what makes the most sense for the success of the product.
+
+| Team | Purpose |
+|:--|:--|
+| Analytics | Analyzing the data we use for gaining insight into Desktop and its users |
+| Core | Internal GitHub team that maintains Desktop |
+| Comrades | Avenue for community members to get more involved in the Desktop project |
+| Docs | Responsible for creating documentation Desktop users can consult when needed |
+| Eng | Responsible for implementing Desktop's new features, fixing bugs, and all other developement tasks |
```

Comment:
Can we rename this to `desktop/engineering`? We've got autocomplete when mentioning teams, we don't need to be succinct here

Reply:
I have a comment below 👇 that applies here as well.